src/components/Discussion/index.jsx

```diff
@@ -1,20 +1,23 @@
-import { useState, useRef } from "react";
+import { useState, useRef, useEffect } from "react";
 import "./index.css";
 import Navbar from "../Navbar";
 import StarRating from "./rating";
 import Card from "./card";
+import Comment from "./editor";
+import { BiUserCircle } from "react-icons/bi";
 
 export default function Discussion() {
   const [comment, setComment] = useState("");
   const [name, setName] = useState("");
   const [count, setCount] = useState(0);
   const [rating, setRating] = useState(0);
   const [commentList, setCommentList] = useState([]);
-  const commentRef = useRef(null);
-  const onClickHandler = (e) => {
+  const [imageUrl, setImageUrl] = useState("");
+  const commentRef = useRef();
+  const ClickHandler = (e) => {
```

Comment:
why it is capitalized? Also what do you mean by ClickHandler, give it a meaningful name